src/Controller/v1/PaymentsController.ts

```diff
@@ -42,7 +42,11 @@ export class PaymentsController extends BaseHttpController {
     await this.httpService.callPaymentsServer(request, response, 'api/extensions', request.body)
   }
 
-  @all('/subscriptions(/*)?')
```

Comment:
by removing this one you will remove quite a few endpoint proxies - have a look at the routes in payments starting with `/subscriptions`: https://github.com/standardnotes/payments/blob/develop/config/routes.rb

Reply:
Sorry, I removed this line involuntarily - the subscriptions method was left hanging without decorator below this. Will re-add it